github/issues_comments.go

```diff
@@ -100,8 +103,13 @@ func (s *IssuesService) GetComment(owner string, repo string, id int) (*IssueCom
 
 // CreateComment creates a new comment on the specified issue.
 //
+// Comments longer than 65536 will return an error.
+//
 // GitHub API docs: http://developer.github.com/v3/issues/comments/#create-a-comment
 func (s *IssuesService) CreateComment(owner string, repo string, number int, comment *IssueComment) (*IssueComment, *Response, error) {
+	if comment != nil && comment.Body != nil && len(*comment.Body) > maxCommentLen {
+		return nil, nil, fmt.Errorf("Comments may not be longer than %d", maxCommentLen)
```

Comment:
Error strings should not be capitalized, see https://github.com/golang/go/wiki/CodeReviewComments#error-strings.

Actually, can we use the same phrasing as what GitHub API returns? According to the PR description, it would be something like:

> body is too long (maximum is 65536 characters)